docs/source/llms/tracing/index.rst

```diff
@@ -706,6 +725,28 @@ and the :py:meth:`mlflow.client.MlflowClient.delete_trace_tag` method to remove
     client.delete_trace_tag(request_id="12345678", key="tag_key")
 
 
+Async Logging
+-------------
+
+By default, MLflow Traces are logged synchronously. This may introduce a performance overhead when logging Traces, especially when your MLflow Tracking Server is running on a remote server. If the performance overhead is a concern for you, you can enable **asynchronous logging** for tracing in MLflow 2.16.0 and later.
+
+To enable async logging for tracing, call :py:func:`mlflow.config.enable_async_logging` in your code. This will make the trace logging operation non-blocking and reduce the performance overhead.
+
+.. code-block:: python
+
+    import mlflow
+
+    mlflow.config.enable_async_logging()
+
+    # Traces will be logged asynchronously
+    with mlflow.start_span(name="foo") as span:
+        span.set_inputs({"a": 1})
+        span.set_outputs({"b": 2})
```

Comment:
Do we need to call `flush_async_logging` to flush pending async logging operations?

Reply:
Not really. The notebook rendering is synchronous so they will see traces as soon as it is generated. There may be a small time gap until the trace is recorded in the backend i.e. Experiment page. Maybe I can add note like "if you don't see the trace in Experiment page, use flush_async_logging API...."

Reply:
> Maybe I can add note like "if you don't see the trace in Experiment page, use flush_async_logging API...."

Let's add it. The current code looks like async code that's missing `await`.